app.js

```diff
@@ -1,71 +1,94 @@
 var userName = prompt("What is your name?");
-alert("Ahoy there " + userName + "! Where be my booty?");
+var pElOne = document.getElementById('a1');
+var pElTwo = document.getElementById('a2');
+var pElThree = document.getElementById('a3');
 
-var playGame = prompt("Just kidding, you don't owe me anything. Would you like to play a game, yes or no?");
-var uCase = playGame.toUpperCase();
 var counter = 0;
-console.log(uCase);
-if (uCase === "YES" || uCase === "Y") {
-  var q1 = prompt("Great! Let's get started! Do you think I'm a cat man?");
-  uCase = q1.toUpperCase();
-  console.log(uCase);
-    if (uCase === "YES" || uCase === "Y") {
-      alert("Damn right I am. And vicious little buggers they are. Never turn your back on them I say!");
-      counter ++
-      console.log(counter);
-    } else {
-      alert("HA! You obviously don't know the caliber of the man you're dealing with. Never trust a man without a cat.");
-    }
 
-    var q2 = prompt("Of my two favorite cardinal sins, which do you think is my most favorite? Gluttony or Sloth?");
-    uCase = q2.toUpperCase();
-    console.log(uCase);
-      if (uCase === "GLUTTONY" || uCase === "SLOTH") {
-        alert("Trick question! They go hand in hand. You can't have gluttony without sloth. I can't at least!");
-        counter++
-        console.log(counter);
-      }
+var els = [pElOne, pElTwo, pElThree];
 
-      var q3 = prompt("Given what you've learned about me so far, can you believe I didn't get married until I was 35?");
-      uCase = q3.toUpperCase();
-      console.log(uCase);
-        while (uCase === "YES" || uCase === "Y") {
-          var q3 = prompt("Very funny. Try again funny person. ");
-          uCase = q3.toUpperCase();
-        }
-      alert("I know! Right?!? So crazy I wasn't snatched up sooner!");
-      counter++
-      console.log(counter);
+var questions [
```

Comment:
Code crashes here. See [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array) for example.